.azure-pipelines/azure-pipelines-build.yml

```diff
@@ -163,7 +163,7 @@ jobs:
             if [ $(dbg_image) == yes ]; then
               make  $BUILD_OPTIONS INSTALL_DEBUG_TOOLS=y target/sonic-$(GROUP_NAME).bin
               mv target/sonic-$(GROUP_NAME).bin target/sonic-$(GROUP_NAME)-dbg.bin
-            elif [ $(ss_image) == yes ]; then
```

Comment:
if dbg;then
fi
if ss_image;then
fi

Reply:
thanks for catching this silly mistake. fixed it